BitwardenShared/Core/Auth/Models/Request/SingleSignOnDomainsVerifiedRequestModel.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import Networking
+
+// MARK: - SingleSignOnDomainsVerifiedRequestModel
+
+/// API request model for getting the single sign on verified domains of a user from their email.
+///
+struct SingleSignOnDomainsVerifiedRequestModel: JSONRequestBody {
+    // MARK: Static Properties
+
+    static let encoder = JSONEncoder()
```

Comment:
🤔 Curious what you think, would it be better to not specify individual encoders/decoders on a per-request/response basis? By default, this will pick up the default encoder:

https://github.com/bitwarden/ios/blob/df31b099c99af73c44a38c2d420372c36f3230e1/BitwardenShared/Core/Platform/Services/API/Extensions/JSONRequestBody%2BBitwarden.swift#L4-L6

Reply:
Totally agree and missed that one; I copy pasted and forgot to remove the line.